src/managers/integration_hub.py

```diff
@@ -35,6 +41,8 @@ def _ssl_enabled(endpoint: str | None) -> str:
     def _s3_conf(self) -> dict[str, str]:
         if s3 := self.s3:
             return {
+                "spark.hadoop.fs.s3a.path.style.access": "true",
+                "spark.eventLog.enabled": "true",
```

Comment:
Do we really want this? Wouldn't this enable event logs automatically on S3 relation? In my view, this configuration is not quite related with S3 integration.

Reply:
Also, if we are adding this configuration, I think `spark.eventLog.dir` would accompany this as well.